### ISSUE TITLE
VO doesn't work properly with home/end keyboard shortcuts

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#295]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.4

**Browser**
safari

**Problem description**
https://github.com/phetsims/qa/issues/702

When VO is on, the home/end (fn + left/right arrow) keys don't work properly. Instead of changing the mass values to min/max, the VO focus moves off of the vertical spinner.  This is also seen in the published sim. 

**Steps to reproduce**
1. Turn on VO
2. Tab to either vertical spinner
3. On a Mac, use fn + right arrow and fn + left arrow to try and change values to 10 and 1. Then use arrow keys to change the values in smaller increments. You will see that the values change because the focus highlight is still on the spinner but VO focus is no longer on that particular spinner.

**Visuals**
https://drive.google.com/file/d/13rgXi0dWbS8eGaujg1rFQBZo7UlGfvkO/view?usp=sharing

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15
Language: en-us
Window: 1440x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by Nancy-Salpepi** [comment]
I noticed that this issue is on the known bugs list so I am closing.

https://github.com/phetsims/resistance-in-a-wire/issues/163
